.gitignore

```diff
@@ -148,3 +148,4 @@ packages/lsp/src/_*
 # generated reports
 webpack-bundle-analyzer.html
 examples/**/test-results/
+docker/start.sh
```

Comment:
This file isn't relevant to this change. Can you please remove this line from `.gitignore`, and remove the empty `docker/start.sh` file from your branch?

Reply:
I will attempt to do so, it was not intended to be in the commit.